Task1/src/showSpeed.kt

```diff
@@ -0,0 +1,7 @@
+class Speed(_speed: Int){
+    val speed = _speed
+
+    fun showSpeed (speed: Int){
+        println("Your speed now $speed ")
```

Comment:
This statement sound's weird.  Add more Grammar: Now speed u'r speed is $speed. Something like this. If u wont write in this case. Why "You"? No in every case it's a speed same object that call this method.